### DISCUSSION TITLE
Custom black color and buttons

### DISCUSSION THREAD

**Discussion by koukalp** [mantinedev/mantine#7169] [Q&A]
I am not sure whether I am experiencing an issue with the library or not. 

Reproduction: 

1. define custom "black" color in the theme:
```
    colors: {
      black: ['red', 'red', 'red', 'red', 'red', 'red', 'red', 'red', 'red']
   }
```

2. use "black" color for a button
```
<Button variant="filled" color="black">Text</Button>
```

=> the button uses #000000 instead of "red"

What's going on?

**Comment by rtivital** [comment]
`black` is referencing `theme.black`, not `theme.colors.black`

**Comment by koukalp** [comment]
weird is my theme has this:
```

   black: 'red',

```

and the buttons still use #000000

**Comment by koukalp** [comment]
and BTW it works as you describe in Flex, Box, Anchor, ... but Button seems to have a special treatment.

**Comment by rtivital** [comment]
If you want to create a custom color, use a different color name, for example `secondary` or `dark-gray`.

**Reply by koukalp** [reply]
ok, that worked of course, it's just an exception to our "contract" with designers and color names in Figma. 
Sounds like Button component really gets some special treatment regarding "black" color. Thanks for your support.
